### PR TITLE
fix(headerbar): upgrade version so it gets latest features and fixes

### PR DESCRIPTION
**Pull request by HendrikThePendric**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,25 +22,30 @@
...
         "material-ui": "^0.20.0",
         "prop-types": "^15.5.10",
-        "react": "^16.3.0",
-        "react-dom": "^16.2.0",
+        "react": "^16.13.0",
+        "react-dom": "^16.13.0",
         "react-router": "^3.2.1",
-        "rxjs": "^5.5.7"
+        "rxjs": "^5.5.7",
+        "styled-jsx": "^3.3.0"
```
Comment (ismay) [inline package.json:36]: I think you can remove this. It's not used directly in the app anywhere it seems.

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+            {
+                "modules": false
+            }
+        ]
+    ],
+    "plugins": [
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-transform-runtime"
+    ]
 }
```
Comment (ismay) [inline .babelrc:15]: I was wondering why the config was changed to these specific settings. If we want to migrate this app to CRA anyway, maybe their preset would be useful for us for example: https://github.com/facebook/create-react-app/tree/master/packages/babel-preset-react-app

File: package.json [hunk 3]
```diff
@@ -22,25 +22,30 @@
...
-        "babel-preset-es2015": "^6.24.1",
-        "babel-preset-react": "^6.24.1",
-        "babel-preset-stage-0": "^6.24.1",
+        "@babel/core": "^7.2.2",
+        "@babel/plugin-proposal-class-properties": "^7.2.3",
+        "@babel/plugin-transform-runtime": "^7.2.0",
+        "@babel/preset-env": "^7.2.3",
+        "@babel/preset-react": "^7.0.0",
+        "@babel/runtime": "7.3.4",
+        "@dhis2/cli-style": "^2.2.2",
```
Comment (ismay) [inline package.json:45]: Are we using this anywhere? If not I'd just omit it.

File: package.json [hunk 4]
```diff
@@ -22,25 +22,30 @@
...
+        "babel-eslint": "^10.0.1",
+        "babel-loader": "^8.0.5",
+        "cross-env": "^5.2.0",
```
Comment (ismay) [inline package.json:48]: Same here. It doesn't seem like we're using this.

File: src/app.router.js [hunk 5]
```diff
@@ -21,7 +22,7 @@ import PasswordChangeSuccessDialog from './account/PasswordChangeSuccessDialog';
 
 function WrAppadApp(props) {
```
Comment (ismay) [inline src/app.router.js:23]: Not your change, but I just wanted to point it out. I had to do a double take when I first read it 😅

File: src/layout/app.css [hunk 6]
```diff
@@ -1,26 +1,23 @@
 
-html {
+html, body {
   background-color: #f3f3f3;
   font-family: Roboto, Arial, sans-serif;
+  padding: 0;
+  margin: 0;
 }
```
Comment (ismay) [inline src/layout/app.css:7]: I guess stylistically I'd prefer to not repeat the bg color and font-family here. But it's minor.

File: package.json (hunk 1 above)
Comment (HendrikThePendric) [reply package.json:36]: the app breaks without this

**Pull request merged**